multegula.go

```diff
@@ -118,6 +118,22 @@ func getMessage(nodes []messagePasser.Node, localNodeName string) messagePasser.
 	return messagePasser.Message{Source: localNodeName, Destination: dest, Content: content, Kind: kind}
 }
 
+/* receive message from PyBridge and send to messagePasser */
+func sendToMessagePasser() {
+    for {
+        message := bridges.ReceiveFromPyBridge()
+        messagePasser.Send(message)
+    } 
+}
+
+/* receive message from MessagePasser and send to PyBridge */
+func receiveFromMessagePasser() {
+    for {
+        message := messagePasser.BlockReceive()
```

Comment:
Thanks for pointing that out. I tried not to use blockReceive(). But since we have a routine in multegula waiting for messages come from MessagePasser in a for loop. If we use receive(), which is not block, rather than using blockReceive(), it will loop very quickly and other routines in multegula will be affected and they even cannot make progress. That's why I created a bolckRecevie() method. When I created receivedQueue() at first, I supposed that it should act as a buffer for received messages. All received messages should be put into this channel before delivered. So that all messages will only be received by Receive() or BlockReceive() method. I'm not sure how you used receivedQueue in reliableMulticast. But if you put messages which is ready for delivering into receivedQueue, they won't be missed and will be received by Receive() or BlockRecieve().